config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,28 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I doubt we need a CSP. Should I just delete it?

Reply:
Delete away.